app/scripts/lib/metrics.js

```diff
@@ -339,6 +339,8 @@ define([
      */
     logScreen: function (screenName) {
       this.logEvent(this.screenToId(screenName));
+      // log the path as a separate path event
+      this.logEvent('path:' + this._window.location.pathname);
```

Comment:
IIRC, A debate debate long ago caused us to convert all `:` to `.` to make querying simpler in Kibana.